integration/env/new_azure_go_file_to_improve_the_diff_should_rplace_azure_go.go

```diff
@@ -0,0 +1,125 @@
+package env
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/giantswarm/azure-operator/integration/network"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+)
+
+var (
+	// TODO Consider if we need to pass it at all.
+	azureCIDR = getEnv("AZURE_CIDR")
+	// TODO Why do we still need all the subnets below?
+	azureCalicoSubnetCIDR = mustGetEnv("AZURE_CALICO_SUBNET_CIDR")
+	azureMasterSubnetCIDR = mustGetEnv("AZURE_MASTER_SUBNET_CIDR")
+	azureVPNSubnetCIDR    = mustGetEnv("AZURE_VPN_SUBNET_CIDR")
+	azureWorkerSubnetCIDR = mustGetEnv("AZURE_WORKER_SUBNET_CIDR")
+
+	azureClientID       = mustGetEnv("AZURE_CLIENTID")
+	azureClientSecret   = mustGetEnv("AZURE_CLIENTSECRET")
+	azureLocation       = mustGetEnv("AZURE_LOCATION")
+	azureSubscriptionID = mustGetEnv("AZURE_SUBSCRIPTIONID")
+	azureTenantID       = mustGetEnv("AZURE_TENANTID")
+
+	azureGuestClientID       = mustGetEnv("AZURE_GUEST_CLIENTID")
+	azureGuestClientSecret   = mustGetEnv("AZURE_GUEST_CLIENTSECRET")
+	azureGuestSubscriptionID = mustGetEnv("AZURE_GUEST_SUBSCRIPTIONID")
+	azureGuestTenantID       = mustGetEnv("AZURE_GUEST_TENANTID")
+
+	// TODO this should be prefixed with AZURE_.
+	commonDomainResourceGroup = mustGetEnv("COMMON_DOMAIN_RESOURCE_GROUP")
```

Comment:
This could be done in `Build()` but I think it's good to have that on the top of the file for visibility. This gives nice overview what variables are there which of them are required and which optional.

Reply:
I think I should put that comment in the code.

Reply:
Comment  added.